src/components/clusters/Clusters.tsx

```diff
@@ -83,12 +83,7 @@ const Clusters: React.FC = () => {
             <ClustersTable rows={clusterRows} deleteCluster={deleteCluster} />
           </PageSection>
           <ClusterWizardToolbar>
-            <ToolbarButton
-              variant={ButtonVariant.primary}
-              component={(props) => <Link to="/clusters/new" {...props} />}
-            >
-              Create New Cluster
-            </ToolbarButton>
+            <NewClusterModalButton ButtonComponent={ToolbarButton} />
```

Comment:
Why do we need this flexibility with button component?

Reply:
You're right, it is not needed here in this case (I've copied it over from discovery Iso modal which is triggered from 2 different buttons.